Makefile

```diff
@@ -78,6 +79,27 @@ gdbstub-test: $(BIN)
 	$(Q).ci/gdbstub-test.sh && $(call notice, [OK])
 endif
 
+ENABLE_JIT ?= 0
+$(call set-feature, JIT)
+ifeq ($(call has, JIT), 1)
+CFLAGS += -I./src/mir
+LDFLAGS += src/mir/libmir.a -lpthread
+OBJS_EXT += compile.o
+
+src/jit_template.c:
+	$(Q)python ./tools/gen-jit-template.py $(EXT_LIST) > $@
```

Comment:
Rewrite as
```shell
$(Q)tools/gen-jit-template.py $(EXT_LIST) > $@
```

You shall specify a proper [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) accordingly. i.e.,
```
#!/usr/bin/env python3
```